src/lib/serviceRoutines/statisticsTreat.cpp

```diff
@@ -240,13 +250,16 @@ std::string renderSemWaitStats(void)
   jh.addFloat("subCache",          semTimeCacheGet());
   jh.addFloat("connectionContext", mutexTimeCCGet());
   jh.addFloat("timeStat",          semTimeTimeStatGet());
+  jh.addFloat("metrics",           ((float) metricsMgr.semWaitTimeGet()) / 1000000);
```

Comment:
Yeah, it's not complete. I would say we just forgot about it.
There are more semaphores as well